deploy/templates/notificationtemplates/appstudio/userprovisioned/notification.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <title>
+    Note: This is an automated email message. Please do not reply to this email. For any assistance, contact us at sscs@redhat.com.
+  </title>
+  <style>
+    a:hover {
+      text-decoration: underline !important;
+    }
+    p {
+      text-align: left;
+      margin: 30px 0;
+    }
+  </style>
+</head>
+
+<body
+        style="
+       padding: 10px;
+       padding: 0;
+       background-color: #f9f9f9;
+       font-family: 'Open Sans', sans-serif;
+       font-size: 15px;
+       font-weight: lighter;
+       line-height: 1.2;"
+>
+<div
+        style="
+       min-height: 300px;
+       max-width: 750px;
+       margin: 0 auto;
+       padding: 20px;
+       border: 1px solid #d7d7d7;
+       border-radius: 4px;
+       background-color: #fff;
+       box-shadow: 0 2px 4px #d7d7d7;"
+>
+
+  <p>
+    Dear xyz,
+  </p>
+
+  <p>
+    Welcome to Red Hat CI/CD! Red Hat CI/CD saves you weeks of time by automating the onboarding process for containerized apps with CI/CD. It also offers granular customization for pipelines, once they are running. And it provides unparalleled security for your software supply chain.
+  </p>
+
+  <p>
+    We have enabled your access, so to get started, you can now visit Red Hat CI/CD console and select + Create an application.
+  </p>
+
+  <p>
+    If you’d rather access Red Hat CI/CD through your CLI, or if you have any questions that are not answered in the web UI, you can also visit our documentation at "dummy-link".
+  </p>
```

Comment:
Let's modify the template to make it closer to https://docs.google.com/document/d/11P8Gj8Q9gindbB9wmbFuIKv53SQL3b99vHUy0jemPfw/edit
```suggestion
  <p>
    Welcome to Red Hat CI/CD! Red Hat CI/CD saves you weeks of time by automating the onboarding process for containerized apps with CI/CD. It also offers granular customization for pipelines, once they are running. And it provides unparalleled security for your software supply chain.
  </p>

  <p>
    We have enabled your access, so to get started, you can now visit {{.RegistrationURL}} and select + <b>Create an application</b>.
  </p>

  <p>
    If you’d rather access Red Hat CI/CD through your CLI, or if you have any questions that are not answered in the web UI, you can also visit our documentation at https://redhat-appstudio.github.io/docs.stonesoup.io/Documentation/main/index.html.
  </p>
```

Reply:
done in [29be8b4](https://github.com/codeready-toolchain/host-operator/pull/754/commits/29be8b41347898468009c261876303dde8b37305)

Reply:
@alexeykazakov I can also add the First name and Last Name as specified in the [doc](https://docs.google.com/document/d/11P8Gj8Q9gindbB9wmbFuIKv53SQL3b99vHUy0jemPfw/edit), since those values are already available [here](https://github.com/codeready-toolchain/toolchain-common/blob/master/pkg/notification/notification_builder.go#L145-L146) by passing [userContext](https://github.com/codeready-toolchain/host-operator/blob/master/controllers/masteruserrecord/sync.go#L225).

Also, just noticed that the content of the email is not exactly same. 

> _**Red Hat CI/CD**_ saves you weeks of time by automating the onboarding process for containerized apps with CI/CD. It also offers granular customization for _**pipelines, once they are running**_. And it provides unparalleled security for your software supply chain.

vs (provided in the doc):
> _**This new platform**_ saves you weeks of time by automating the onboarding process for containerized apps with CI/CD. It also offers granular customization for _**build pipelines**_. And it provides unparalleled security for your software supply chain.

Should I make the change as expected in the doc or keep the change? (Personally i don't mind the difference, and can follow up with the HAC team if we keep the current changes.)